antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/atn/ParserATNSimulator.kt

```diff
@@ -279,15 +279,6 @@ public open class ParserATNSimulator(
   protected var _outerContext: ParserRuleContext? = null
   protected var _dfa: DFA? = null
 
-  /**
```

Comment:
Is this not useful for testing in our case?

Reply:
Not really, or at least not now. We'll restore it keeping in mind the nullability when it will be required.